.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,85 @@
+name: firebase deploy
+
+on:
+  push:
+    paths:
+      - "_src/**"
+      - ".github/workflows/nodejs.yml"
+      - "package.json"
```

Comment:
Should we add `bower.json` to this? I didn't think about this when reviewing raster but that's probably more important than package.json.

Reply:
we certainly can.